src/App.jsx

```diff
@@ -78,7 +78,7 @@ class App extends React.Component {
     const { history } = this.props;
     if (accountData && accountData.acc) {
       await ProtonSDK.logout();
-      this.setState({ actor: '', accountData: {}, session: '' });
+      this.setState({ actor: '', accountData: {}, session: '', error: '' });
```

Comment:
Marking as unapproved, and a comment is necessary to change the PR review state.
We should refactor to handle error clearing in a more global way, rather than case by case as we discussed on slack.